test/models/localization_files_test.rb

```diff
@@ -144,7 +144,7 @@ def source_files(*paths, &block)
 
   def test_api_error_translations
     tags = []
-    file = "#{::Rails.root}/app/classes/api/errors.rb"
+    file = "#{::Rails.root}/app/classes/api2/errors.rb"
```

Comment:
@pellaea: Should we run this for both `.../api/errors` and `.../api2/errors? 
If not, this looks right.
(And I think it explains something that puzzled me: How did we end up with the missing translation that @nimmolo fixed in #995?)

Reply:
Ahaa!